ibis/backends/duckdb/__init__.py

```diff
@@ -1566,6 +1566,45 @@ def to_geo(
         with self._safe_raw_sql(copy_cmd):
             pass
 
+    @util.experimental
+    def to_json(
+        self,
+        expr: ir.Table,
+        path: str | Path,
+        *,
+        compression: Literal["auto", "none", "gzip", "zstd"] = "auto",
+        dateformat: str | None = None,
+        timestampformat: str | None = None,
+    ) -> None:
+        """Write the results of `expr` to a json file of [{column -> value}, ...] objects.
+
+        This method is eager and will execute the associated expression
+        immediately.
+        See https://duckdb.org/docs/sql/statements/copy.html#json-options
+        for more info.
+
+        Parameters
+        ----------
+        expr
+            The ibis expression to execute and persist to Delta Lake table.
+        path
+            URLs such as S3 buckets are supported.
+        compression
+            Compression codec to use. One of "auto", "none", "gzip", "zstd".
+        dateformat
+            Date format string.
+        timestampformat
+            Timestamp format string.
+        """
+        opts = f", COMPRESSION '{compression.upper()}'"
+        if dateformat:
+            opts += f", DATEFORMAT '{dateformat}'"
+        if timestampformat:
+            opts += f", TIMESTAMPFORMAT '{timestampformat}'"
```

Comment:
Eventually this should be done with sqlglot as much as possible, but this is fine for now.

Reply:
Ah I see, thanks!